stringutils.go

```diff
@@ -70,3 +74,30 @@ func hashsum(s string, hasher hash.Hash) string {
 	hasher.Write([]byte(s))
 	return hex.EncodeToString(hasher.Sum(nil))
 }
+
+func foundInHIBP(s string) int {
+	bytes := []byte(s)
+	h := sha1.New()
+	h.Write(bytes)
+	result := hex.EncodeToString(h.Sum(nil))
+
+	firstFive := result[0:5]
+	restOfHash := strings.ToUpper(result[5:])
+
+	url := "https://api.pwnedpasswords.com/range/" + firstFive
+
+	resp, err := http.Get(url)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
This method should return an (additional) error, so we can return whatever error occurred to the library's user. We should never call `log.Fatal` or `panic` in a library, since this prevents the library's user to handle the error in a reasonable way.